forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt

```diff
@@ -24,6 +35,13 @@ data class ForageConfig(
 interface ForageElement<T : ElementState> {
     var typeface: Typeface?
 
+    /**
+     * Sets the necessary [ForageConfig] configuration properties for a [ForageElement].
+     *
+     * [setForageConfig] must be called before any other methods can be executed on the Element.
+     *
+     * @param params A [ForageConfig] instance that specifies a `merchantId` and `sessionToken`.
```

Comment:
In this case the parameter is called `forageConfig`, as seen in the line below
```suggestion
     * @param forageConfig A [ForageConfig] instance that specifies a `merchantId` and `sessionToken`.
```